deepchecks/core/check_result.py

```diff
@@ -161,9 +161,9 @@ def have_display(self) -> bool:
         """Return if this check has display."""
         return bool(self.display)
 
-    def passed_conditions(self) -> bool:
+    def passed_conditions(self, fail_on_warning=True) -> bool:
```

Comment:
IMO fail on warning should be default False

Reply:
@noamzbr and @shir22 thinks otherwise